From b06e34de1689c123ac35bad562aadb9a02af28c8 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Tue, 10 Oct 2023 20:53:25 +0200
Subject: [PATCH 68/69] Revert "maple_tree: use mas_node_count_gfp on
 mas_expected_entries"

This reverts commit 1ad2eff087260b9edbc8e080aa780a918961758f.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 lib/maple_tree.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/maple_tree.c b/lib/maple_tree.c
index ab47b2a2f..1c4629ab9 100644
--- a/lib/maple_tree.c
+++ b/lib/maple_tree.c
@@ -5668,7 +5668,7 @@ int mas_expected_entries(struct ma_state *mas, unsigned long nr_entries)
 	/* Internal nodes */
 	nr_nodes += DIV_ROUND_UP(nr_nodes, nonleaf_cap);
 	/* Add working room for split (2 nodes) + new parents */
-	mas_node_count_gfp(mas, nr_nodes + 3, GFP_KERNEL);
+	mas_node_count(mas, nr_nodes + 3);
 
 	/* Detect if allocations run out */
 	mas->mas_flags |= MA_STATE_PREALLOC;
-- 
2.42.0

